src/Lightning/NoiseProtocol.Test/StringUtilities.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NoiseProtocol.Test
+{
+    public static class StringUtilities
```

Comment:
maybe we need to unify this somewhere its duplicated too much

Reply:
Will do as part of another task in master